frontend/src/components/Dashboard.vue

```diff
@@ -1,24 +1,49 @@
 <template>
-  <template
-    v-if="loading"
-  >
+  <template v-if="loading">
     <Spinner />
   </template>
 
-  <div
-    v-if="!loading"
-    id="content-container"
-  >
-    <template
-      v-for="content in contents"
-      :key="content"
+  <template v-if="!loading">
+    <div class="content-container">
+      <template
+        v-for="content in visibleContent"
+        :key="content"
+      >
+        <component
+          :is="contentDisplayer"
+          :content="content"
+          :hidden="isHidden(content)"
+          @toggle-visibility="toggleVisibility"
+        />
+      </template>
+    </div>
+    <div
+      v-if="hiddenContent.length > 0"
+      class="hidden-elements"
     >
-      <component
-        :is="contentDisplayer"
-        :content="content"
-      />
-    </template>
-  </div>
+      <hr>
+      {{ hiddenContent.length }} hidden jobs
```

Comment:
Dashboard is a generic component, personally I'll prefer not to hard code like **hidden jobs**

Reply:
Solved